fastlane/Fastfile

```diff
@@ -342,7 +342,8 @@ platform :android do
         **UPLOAD_TO_PLAY_STORE_COMMON_OPTIONS,
         aab: aab_artifact_path,
         track: track,
-        release_status: 'draft'
+        release_status: 'inProgress',
+        rollout: is_beta ? '0.20' : '0.10' # Start at 20% for betas, 10% for Production
```

Comment:
I'm not sure if this covers that but we start our prod releases at 1% on Fridays, then on Mondays we bump to 10% and rollout from there.

Reply:
@MiSikora  Thanks for asking. I saw that from the MC scenario, but is 1% really ever in effect in practice though?

Because what I read from the scenario is that:
 - You **_submit for review_** with a rollout of 1% on Friday…
 - But once it's approved by Google, you update that rollout value to 10% on Monday on the moment that you release the approved version?

Or am I misreading the current scenario? Maybe the current scenario is missing a task about «Click "Publish N changes» at the start of Monday, before the currently-first task of Monday that is about updating the rollout to 10%, meaning that you start early Monday morning by clicking the button to publish to 1% and only later on that same Monday you update the rollout?

Reply:
I believe we've always submitted for publication at 1% on Fridays. At least, that's what I’ve been doing, and from my memory, others have as well, since I’ve received emails about live rollouts over the weekend. I think publishing at 1% over the weekend isn’t a bad idea. The volume is extremely low, so the risk is minimal, and it gives us a chance to catch any crashes that might occur during that time.

That said, I’m open to changing our publishing workflow to what you described, as it seems like that’s how it should have been. 😓

Reply:
I'm not sure I'm following. When you submit on Friday, you only submit to Google for Review.

The way "Managed Publishing" works in Google Play is that you submit for review, then Google reviews and (hopefully) approve your submission… but doesn't publish it yet, and it's only when you go to the "Publishing Overview" page after they approved it and explicitly click on the "Publish N changes" button there that the rollout actually starts.

The rollout does not start immediately after Google approves the submission, but only when you explicitly click on the "Publish Changes" button.

Hence why I'm saying that having the rollout set to 1% during the weekend does not do anything (unless Google speed-approves your submission on Friday before your Friday EOD and you decide to click the "Publish Changes" on Friday before the weekend instead of waiting for Monday to do that… but I don't think you're doing that either)

Reply:
I'm saying I, and from what I can tell we as a team, haven't been using Managed Publishing when submitting on Fridays.

Reply:
You can't choose to use Managed Publishing or not on a case by case basis.

Managed Publishing is turned on for PocketCasts in Play Store, and that affects all submissions (and you can't turn it off unless you ask Systems as you need special permissions for that iinm, and that would affect all the builds anyway, it's an all-or-nothing setting in Play Store Console)
<img width="212" alt="image" src="https://github.com/user-attachments/assets/cb74e11c-d1d8-402c-b27c-5278d1394e0b">

So given Managed Publishing is turned on, this has always worked like described in [this Google help page](https://support.google.com/googleplay/android-developer/answer/9859654?hl=en).

When you submit for review on Friday, the submitted build will be in "Changes submitted for Review" section of the "Publishing Overview" page, and Google will review them at some point over the weekend, but the rollout won't start until you explicitly click "Publish N changes" for those approved changes on Monday. So nothing is ever rolled out during the weekend, because Managed Publishing is on in our Play Store Console settings for the whole project.

Reply:
Sorry, but we're going in circles. I definitely can turn managed publishing on and off, and I'm sure that others can as well since we've been doing it in the past. Just from the rollout history:

* `7.77` - Saturday, November 23rd at 1%
* `7.76` - Saturday, November 9th at 1%
* `7.75` - Saturday, October 26th at 1%
* `7.74` - Monday, October 14th at 1%
* `7.73` - Saturday, September 28th at 1%

Reply:
Wait if you've actively been turning Managed Publishing on and off on the PlayStore (turning it off on Fridays, and remembering to turn it back on on Monday, manually each time?!) then I'm confused because it's documented nowhere in the MC scenario that you'd have to do that manually on each Friday and Monday 🤔

Reply:
I haven't. It's been off by default when I've been publishing. I remember turning it off once when we had to coordinate a release of one feature with a partner.

Reply:
Ok. I think you're the only Android product in a8c that is not using Managed Publishing all the time then 😅 

I'm still confused why it's not explicitly documented in the release scenario that one would have to decide to turn it on or off; especially given right now it's turned ON (just like for all our other a8c products in Google Play) and I'd have assumed you don't keep changing that setting constantly.

Or is it usually _always_ turned OFF, and the Release Manager is not supposed to constantly change it, and I was just unlucky to check today at a coincidental time that was just after you recently decided to temporarily turn it ON just for this sprint?

If tomorrow I'm covering for Release Management of PCAndroid, what am I supposed to do on Fridays then? Am I supposed to submit for review but also turn Managed Publishing off? And then remember to turn in back ON on Monday? Or is it supposed to always be turned OFF and I'm not supposed to touch it (and the fact that it's ON right now is just a temporary glitch)?!

Reply:
Updated the automation to use 1% instead in [2f18528](https://github.com/Automattic/pocket-casts-android/pull/3333/commits/2f1852882c6d3e3113d494ee968c1fe699ceab67) btw; I'll rework the changes I was working on for the MC scenario accordingly.

Reply:
> especially given right now it's turned ON

It's not. This is what I see in the Publishing Overview.

![Screenshot 2024-12-05 at 20 40 23](https://github.com/user-attachments/assets/db7e6976-1289-412b-a600-ca7b715dfc34)

I'm not sure how to answer the rest of your questions. I haven't been juggling Managed Publishing. It's off whenever I start the publishing process. I can't guarantee what others have been doing but I'm 99% sure they haven't been juggling it as well. Maybe it happens one in a while like with `7.74` which got released on October 14th but was submitted on October 11th? I can't really say what's been going on.

To reiterate I haven't turned managed publishing on or off during the release. I remember doing it only once when when we needed to coordinate things for one of the features. And If I recall correctly it was done for a beta release.

Reply:
Ooof ok  I think I need to get some rest  🤦 because I was sure I checked and saw it ON… but now I've rechecked and it's indeed OFF for PCAndroid… So I guess I was too tired by my day and probably accidentally checked the GPC page for the wrong app or something when I saw it being ON…  🤦

My bad, and thanks for insisting and clarifying.

If it's always been OFF all that time for PCAndroid that clarifies things a lot, you can ignore my above questions then, sorry 😓